### DISCUSSION TITLE
Possible to use AMD Integrated Graphics For Hardware Acceleration?

### DISCUSSION THREAD

**Discussion by NickM-27** [blakeblackshear/frigate#2544] [Ask A Question] [answered]
Hi, I have a NAS with AMD 3200G, running frigate in a docker container. It is an ITX system so I don't have much option to use a GPU as the single PCIe slot is already taken up. 

I was wondering if there are any tips to get the integrated graphics from the AMD cpu to be used for hardware acceleration? I tried using similar to intel and the AMD GPU but none of the recommended ones worked, camera became unviewable and ffmpeg threw errors. Just wanted to see if anyone else has gotten this working?

**Comment by NickM-27** [comment, marked as answer]
User error, I didn't enable the amdgpu drivers in the unraid setup. To do so follow this guide: https://unraid.net/blog/unraid-6-9-beta35

**Reply by peter247** [reply]
So it looks like you have got it working ?.
But can't say I've found that hardware acceleration makes that much difference , on my 5700g it's saving only about 1/3 of the CPU looking at ffmpeg usage with top.
I guess it would be more if you had to transcode / decode the screams.

**Reply by NickM-27** [reply]
Yep, I agree it wasn't a huge difference but like I said not using it for anything else so might as well take advantage of it.
